eZ/Publish/Core/FieldType/EmailAddress/Type.php

```diff
@@ -201,7 +201,7 @@ public function fromHash($hash)
     public function toHash(SPIValue $value)
     {
         if ($this->isEmptyValue($value)) {
-            return null;
+            return '';
```

Comment:
you'll need to adjust fromHash as well for this, which implies we break the hash format unless we accept both.

Which leads me to: Why is `null` wrong?